### PR TITLE
17 lab david

### PR DESCRIPTION
**Pull request by dstoll243**

### CODE REVIEW

File: src/__test__/profile-router.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            firstName: 'David',
+            lastName: 'Stoll',
+          });
+      })
+      .then(Promise.reject)
+      .catch((error) => {
+        expect(error.status).toEqual(400);
+      });
+  });
+  describe('GET /profiles', () => {
```
Comment (JudyVue) [inline src/__test__/profile-router.test.js:50]: For next time, either don't include the test, or comment out the entire test starting with the `test` block because remember, an empty test block with no `expect` statements gives a false positive.

File: src/model/profile.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const profileSchema = mongoose.Schema({
```
Comment (JudyVue) [inline src/model/profile.js:5]: Won't dock points since the lab didn't explicitly say it, but try not to use the same schemas demo'd in class.

File: README.md [hunk 3]
```diff
@@ -1,50 +1,34 @@
...
+- 200 for a successful get.
+- 400 for a bad request.
+
+## Profile Routes
+
+`profileRouter.post('/profiles', bearerAuthMiddleware, jsonParser, (request, response, next))`: Allows a new profile instance to be posted which provides the request.body of the new profile. Possible responses include: 
+
+- 200 for a successful post
+- 400 for an invalid request
+
```
Comment (JudyVue) [inline README.md:34]: Instructions say the following for your README:

```
In the README.md write documention for starting your server and making requests to each endpoint it provides.
```

Tell me how to start and stop my server or run my tests!
